tests/integration-tests/tests/schedulers/test_slurm/test_slurm_memory_based_scheduling/pcluster.config.update_scheduling.yaml

```diff
@@ -0,0 +1,30 @@
+Image:
+  Os: {{ os }}
+HeadNode:
+  InstanceType: {{ instance }}
+  Networking:
+    SubnetId: {{ public_subnet_id }}
+  Ssh:
+    KeyName: {{ key_name }}
+  Iam:
+    AdditionalIamPolicies:
+      - Policy: arn:aws:iam::aws:policy/AmazonSSMManagedInstanceCore
```

Comment:
Why do we need this policy?

Reply:
I just copied the test case for `test_slurm_config_update` and adapted it to test the updatability of `EnableMemoryBasedScheduling`. I guess it was there for a reason.

Reply:
This policy is necessary for internal reporting. Checked together with @gmarciani. Resolving.